src/main/java/frc/robot/Constants.java

```diff
@@ -13,7 +13,32 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+  public static final double DRIVE_MAX_VELOCITY_METERS_PER_SECOND = .5;
+  public static final double DRIVE_MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = .5;
+  public static final double DRIVETRAIN_TRACKWIDTH_METERS = 0.495;
+  public static final double DRIVETRAIN_WHEELBASE_METERS = 0.5461;
```

Comment:
Did we ensue that the dimensions are measured and units are converted?
Perhaps use Units.inchesToMeters instead of adding a preconverted valeu